src/one_dragon/base/operation/operation.py

```
@@ -33,60 +33,49 @@
 if TYPE_CHECKING:
     from one_dragon.base.operation.one_dragon_context import OneDragonContext
 
-class PreviousNodeStateProxy:
+class NodeStateProxy:
     """
-    一个代理类，用于安全、便捷地访问上一个节点的静态信息和动态执行结果。
+    一个代理类，用于安全、便捷地访问节点的静态信息和动态执行结果。
```

Ruff (0.14.4)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)